index.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html>
+
+<head>
+  <title>Calculator Benchmark</title>
+  <meta charset="utf-8">
+  <meta name="description" content="calculator">
+  <meta name="author" content="Keith Oppel">
+  <link type="text/css" rel="stylesheet" href="main.css">
+</head>
+
+<body>
+  <div class="calculator-container">
+    <div class="calculator-display">
+      <! -- This is a placeholder for the calculator display -->
+    </div>
+    <div class="calculator-row">
+      <div class="calculator-button"><button value="AC">AC</button></div>
```

Comment:
By changing each of the .calculator-button divs to span, and removing display: flex, I was able to get the numbers to arrange in the way that you intended.